functions.php

```diff
@@ -127,54 +127,25 @@ function genesis_sample_responsive_menu_settings() {
 
 }
 
-// Adds support for HTML5 markup structure.
-add_theme_support(
-	'html5',
-	array(
-		'caption',
-		'comment-form',
-		'comment-list',
-		'gallery',
-		'search-form',
-	)
+$config_loader = new Genesis_Config_Getter(
```

Comment:
@nathanrice Should this global var be prefixed? (Perhaps `$genesis_sample_config` or similar.) PHPCS warns:

```shell
47 | ERROR | Variables defined by a theme/plugin should start with
    |       | the theme/plugin prefix. Found: "$config_loader".
    |       | (WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedVariableFound)
```